src/main/java/com/rackspace/ceres/app/services/MetadataService.java

```diff
@@ -76,6 +81,16 @@ public MetadataService(ReactiveCqlTemplate cqlTemplate,
     redisHit = meterRegistry.counter("seriesSetHash.redisCache", "result", "hit");
     redisMiss = meterRegistry.counter("seriesSetHash.redisCache", "result", "miss");
     this.appProperties = appProperties;
+
+    // use GROUP BY since unable to SELECT DISTINCT on primary key column
+    getTenantQuery = "SELECT tenant FROM metric_names GROUP BY tenant";
```

Comment:
Why do we put these here and not in DataTableStatements class?

Reply:
I am thinking to put these metadata tables queries from MetadataService and MetricDeletionService into a separate class so that we would have different classes for DataTable and MetadataTable Statements. @rohitsngh27 , @matsboman and @harageth , your input on this?

Reply:
I think the queries should be in the class where they are executed.

Reply:
@matsboman Should we then create this variable as part of constructor or make it as static final string?

Reply:
static final string I think, that's what we used to do in AT&T and I have seen this practice elsewhere too.

Reply:
taken care.